tests/integration_tests/with_db/deployment/test_language_container_deployer.py

```diff
@@ -2,28 +2,59 @@
 from pathlib import Path
 
 from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
+from exasol_script_languages_container_tool.lib.tasks.export.export_info import ExportInfo
 from pyexasol import ExaConnection
 from pytest_itde import config
 
 from exasol_transformers_extension.deployment.language_container_deployer \
     import LanguageContainerDeployer
-from tests.utils.db_queries import DBQueries
 from tests.utils.parameters import bucketfs_params
 from tests.utils.revert_language_settings import revert_language_settings
 
 
-@revert_language_settings
-def _call_deploy_language_container_deployer(
-        language_alias: str,
-        schema: str,
+def test_language_container_deployer(
+        request,
+        export_slc: ExportInfo,
         pyexasol_connection: ExaConnection,
         bucketfs_config: config.BucketFs,
-        container_path,
-        language_settings):
+):
+    test_name: str = request.node.name
+    schema = test_name
+    language_alias = f"PYTHON3_TE_{test_name.upper()}"
+    container_path = Path(export_slc.cache_file)
+    with revert_language_settings(pyexasol_connection):
+        create_schema(pyexasol_connection, schema)
+        call_language_container_deployer(container_path=container_path,
+                                         language_alias=language_alias,
+                                         pyexasol_connection=pyexasol_connection,
+                                         bucketfs_config=bucketfs_config)
+        result = create_and_run_test_udf(pyexasol_connection, language_alias)
+        assert result[0][0]
+
+
+def create_schema(pyexasol_connection: ExaConnection, schema: str):
     pyexasol_connection.execute(f"DROP SCHEMA IF EXISTS {schema} CASCADE;")
     pyexasol_connection.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
 
-    # call language container deployer
+
+def create_and_run_test_udf(pyexasol_connection: ExaConnection, language_alias: str):
+    pyexasol_connection.execute(textwrap.dedent(f"""
+        CREATE OR REPLACE {language_alias} SCALAR SCRIPT "TEST_UDF"()
+        RETURNS BOOLEAN AS
+
+        def run(ctx):
+            return True
+
+        /
+        """))
+    result = pyexasol_connection.execute('SELECT "TEST_UDF"()').fetchall()
+    return result
```

Comment:
This should always return the same result. You could just
* check the value with `assert result[0][0]`
* don't return the value
* rename the function to `assert_udf_running`